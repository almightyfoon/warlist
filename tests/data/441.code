c8501b_-0q5B4B5l4F5j5j5k4S5s5ocp4U4W4U4W___-0p5C4Q5mcn4Hg4g45gc95fi7