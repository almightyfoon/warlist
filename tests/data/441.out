https://conflictchamber.com/#c8501b_-0q5B4BmJjE5j5j5k4S5s5ocp4U4W4U4W___-0p5C4QjGcn4Hg4g45gmV5fi7

Circle Army - 72 / 75 points
[Theme] The Wild Hunt

[Grayle 1] Grayle the Farstrider [+29]
 - Gorax Rager [7]
 - Warpwolf Stalker [17]
 - Woldwatcher [8]
War Wolf [2]
War Wolf [2]
Wolf Lord Morraig [8]
Reeves of Orboros (max) [16]
 - Reeve of Orboros Chieftain & Standard [0(4)]
Warpborn Skinwalkers (max) [15]
 - Warpborn Skinwalker Alpha [0(4)]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [0(4)]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [4]

Circle Army - 74 / 75 points
[Theme] The Bones of Orboros

[Baldur 2] Baldur the Stonesoul [+30]
 - Megalith [20]
 - Wold Guardian [16]
 - Woldwrath [37]
 - Woldwyrd [9]
Blackclad Stoneshaper [0(3)]
Blackclad Stoneshaper [0(3)]
Blackclad Wayfarer [0(4)]
Gallows Grove [0(2)]
Shifting Stones [3]
Celestial Fulcrum [19]
